.github/workflows/ci.yml

```diff
@@ -247,12 +240,25 @@ jobs:
       - build-pikepdf-wheel
       - build-frontend
     steps:
+      -
+        name: Check pushing to Docker Hub
+        id: docker-hub
+        # Only push to Dockerhub from the main repo
+        # Otherwise forks would require a Docker Hub account and secrets setup
+        run: |
+          if [[ ${{ github.repository }} == "paperless-ngx/paperless-ngx" ]] ; then
```

Comment:
I'll start a discussion here: Do we want to push feature branches to the Docker Hub? Or perhaps only tags/dev/beta and main?

Reply:
Similar to GHCR, I don't think Docker Hub has a image or size limit. Having feature branches available there would help in testing. We'd only have to ask people to change their tags, rather than the entire registry.

Reply:
Works for me!